src/fmgc/src/guidance/vnav/AtmosphericConditions.ts

```diff
@@ -22,7 +28,11 @@ export class AtmosphericConditions {
     update() {
         this.ambientTemperatureFromSim = SimVar.GetSimVarValue('AMBIENT TEMPERATURE', 'celsius');
         this.altitudeFromSim = SimVar.GetSimVarValue('INDICATED ALTITUDE', 'feet');
-        this.casFromSim = this.computeCasFromTas(this.altitudeFromSim, SimVar.GetSimVarValue('AIRSPEED TRUE', 'knots'));
+        this.tasFromSim = SimVar.GetSimVarValue('AIRSPEED TRUE', 'knots');
+        this.casFromSim = this.computeCasFromTas(this.altitudeFromSim, this.tasFromSim);
+        // TODO filter?
```

Comment:
Still relevant?

Reply:
Yeah, although I think we should move all of the navigation stuff out to a navigation module that takes in data from ADIRUs etc. so I won't do it now.